build/cqs.tsv

```diff
@@ -8,10 +8,10 @@ com.clearspring.analytics:stream	2.9.2	compile
 com.conversantmedia:disruptor	1.2.13	compile
 com.esotericsoftware:kryo-shaded	3.0.3	compile
 com.esotericsoftware:minlog	1.3.0	compile
-com.fasterxml.jackson.core:jackson-annotations	2.9.10	compile
-com.fasterxml.jackson.core:jackson-core	2.9.10	compile
-com.fasterxml.jackson.core:jackson-databind	2.9.10.4	compile
-com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.9.10	compile
+com.fasterxml.jackson.core:jackson-annotations	2.12.1	compile
+com.fasterxml.jackson.core:jackson-core	2.12.1	compile
+com.fasterxml.jackson.core:jackson-databind	2.12.1	compile
+com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.12.1	compile
```

Comment:
I feel like GeoServer and/or Spark uses these Jackson libraries.  Have you tried this PR with either Scala version in GeoServer yet?

Reply:
i believe so, but it's been a while haha. i think these are not shipped into geoserver. the issue was with the spark unit tests dying on the older jackson versions for some reason, even though the spark version did not change.